TerminalDocs/tutorials/shell-integration.md

```diff
@@ -216,13 +216,103 @@ PROMPT $e]133;D$e\$e]133;A$e\$e]9;9;$P$e\%PROMPT%$e]133;B$e\
 
 ### Bash
 
-You can add the following to the end of your `~/.bashrc` to enable shell integration in bash:
+You may source the following script to an active shell with `source` or `.`
+built-in `bash` commands or add it to the end of your `${HOME}/.bash_profile`
+(for login shells) or `${HOME}/.bashrc` (for non-login shells) to enable a
+complete shell integration in `bash` with versions greater or equal to
+`bash-4.4` (where the `PS0` built-in variable was implemented initially).
+Complete shell integration means that every announced terminal feature works as
+designed.
+
+Please note that if there are `PROMPT_COMMAND`, `PS0`, `PS1` or `PS2` variables
+already assigned to any non-default values that may lead to unpredictable
+results. It would be better to test the script with the "clean" shell first by
+executing `env --ignore-environment bash --noprofile --norc` and sourcing the
+described file as noted earlier.
```

Comment:
Nice note. I think this would look nice if we added some styling here like this:
https://github.com/MicrosoftDocs/terminal/blob/9873ef6f661ed0bab5da0a1fb863d6e9fe5e1f9d/TerminalDocs/customize-settings/appearance.md?plain=1#L48-L49
![{31CB54C6-8473-4D51-85E6-D7561AC3C912}](https://github.com/user-attachments/assets/01d88570-21d1-4bad-9044-8a2d134a8010)

or this:
https://github.com/MicrosoftDocs/terminal/blob/9873ef6f661ed0bab5da0a1fb863d6e9fe5e1f9d/TerminalDocs/customize-settings/startup.md?plain=1#L42-L43
![{AA936F15-482A-4F2C-82B9-81E92915E590}](https://github.com/user-attachments/assets/45e075d3-cb04-4ee5-a69a-929ad7e2b6d8)